src/BuildIntegration/BuildFrameworkNativeObjects.proj

```diff
@@ -6,6 +6,8 @@
     <IlcMultiModule>true</IlcMultiModule>
     <NativeIntermediateOutputPath Condition="'$(FrameworkObjPath)' != ''">$(FrameworkObjPath)\</NativeIntermediateOutputPath>
     <BuildingFrameworkLibrary>true</BuildingFrameworkLibrary>
+    <Optimize Condition="'$(Configuration)' == 'Release' and '$(Optimize)' == ''">true</Optimize>
```

Comment:
Why is it better to fill in the defaults here and not pass it in explicitly when invoking msbuild?

Reply:
It's what we do elsewhere where we bypass the source compilation step and inject ourselves into the ILC step. E.g. here:

https://github.com/dotnet/corert/blob/master/tests/CoreCLR/Test.csproj#L9

Reply:
(I think the property normally gets set somewhere with the source compilation step.)